1904-11-03.xml

```diff
@@ -0,0 +1,2652 @@
+                                <cell cols="2">THE SUN</cell>
+                            </row>
+                            <row>
+                                <cell>Rises a.m.</cell>
+                                <cell>Sets p.m.</cell>
+                            </row>
+                            <row>
+                                <cell>Nov. 1</cell>
+                                <cell>Last Quarter</cell>
+                                <cell>1.13 a.m.</cell>
+                                <cell>6.9 a.m.</cell>
+                                <cell>5.9 p.m.</cell>
+                            </row>
+                            <row>
+                                <cell> 7</cell>
+                                <cell>New Moon</cell>
+                                <cell>5.37 p.m.</cell>
+                                <cell>6.13 a.m.</cell>
+                                <cell>5.5 p.m.</cell>
+                            </row>
+                            <row>
+                                <cell>5</cell>
+                                <cell>1st Quarter</cell>
+                                <cell>2.36 a.m.</cell>
+                                <cell>6.20 a.m.</cell>
+                                <cell>4.59 p.m.</cell>
+                            </row>
+                            <row>
+                                <cell>23</cell>
+                                <cell>Full Moon</cell>
+                                <cell>5.11. a.m.</cell>
+                                <cell>6.26 a.m.</cell>
+                                <cell>4.53 p.m.</cell>
+                            </row>
+                            <row>
+                                <cell>30</cell>
+                                <cell>Last Quarter</cell>
+                                <cell>9.38 a.m.</cell>
+                                <cell>6.33 a.m.</cell>
+                                <cell>4.55 p.m.</cell>
+                            </row>
+                        </table>
+                        <p>THE PLANETS,</p>
+                        <p>Mercury is too near the Sun in the early part of the month to be visible,
+                            becoming an evening star later. </p>
+                        <p>Venus is an evening star setting about two hours after the Sun. </p>
+                        <p>Mars rises about 2.30 a.m. and shines as a morning star. Jupiter is to be
+                            seen in the South about 10 p.m. and sets. about 4 p.m. </p>
+                        <p>Saturn souths about 7 p.m. and sets at midnight. </p>
+                        <p>The chief constellations in the south at 9 p.m. are Andromeda and Pegasus
+                            near the zenith, Cetus about 50 deg, and Sculptor about 30 deg. above
+                            the horizon. </p>
+                    </div>
+
+                </div>
+
+            </div>
```

Comment:
Delete this line and the following two lines. They are closing and reopening the page 2 div.

Reply:
If I delete them, it gives me a red box.